### ISSUE TITLE
Fail to execute GS script

### ISSUE THREAD

**Issue by thethlaing-mmds** [spatie/pdf-to-image#116]
Having the following issue with the latest linux, imageick and gs and the code was working fine till gs auto update itself. 

php version is 7.2

`FailedToExecuteCommand `'gs' -sstdout=%stderr -dQUIET -dSAFER -dBATCH -dNOPAUSE -dNOPROMPT -
dMaxBitmap=500000000 -dAlignToPixels=0 -dGridFitTT=2 '-sDEVICE=pngalpha' -dTextAlphaBits=4 -dGraphicsAlphaBits=4 '-r144x144' -dFirstPage=1 -dLastPage=1 '-sOutputFile=/tmp/magick-19207ziBGob4E6t7W%d' '-f/tmp/magick-19207Gum2Xw9QKD6A' '-f/tmp/magick-19207sU6WmPfGlO5e'' (-1) @ error/delegate.c/ExternalDelegateCommand/462`

**Comment by cuparyk** [comment]
Same here.

Ubuntu 16.04.2
GPL Ghostscript 9.26
PHP 7.2

**Comment by thethlaing-mmds** [comment]
It was the issue from GhostScript. 

Fixed it temporarily by downgrading to GS 9.25. 

If you don't know how to do, use the following script. 

```
cd ~
wget https://github.com/ArtifexSoftware/ghostpdl-downloads/releases/download/gs925/ghostscript-9.25-linux-x86_64.tgz

tar -zxvf ghostscript-9.25-linux-x86_64.tgz

cd ghostscript-9.25-linux-x86_64
sudo cp gs-925-linux-x86_64 /usr/local/bin/gs
sudo chmod +x /usr/local/bin/gs
```

Then use ```gs --version``` to verify if it is 9.25 already. 

Still looking forward to the proper fix though.
